web/ui/react-app/src/pages/graph/Panel.tsx

```diff
@@ -117,21 +118,20 @@ class Panel extends Component<PanelProps & PathPrefixProps, PanelState> {
     let path: string;
     switch (this.props.options.type) {
       case 'graph':
-        path = '/api/v1/query_range';
+        path = `query_range`;
```

Comment:
Maybe revert to normal single quotes now that there is no interpolation happening anymore in this string. Same below.